features/step_definitions/timehelpers_steps.rb

```diff
@@ -1,20 +1,31 @@
+# frozen_string_literal: true
+
+After do
+  travel_back
+end
+
+def no_nest_travel_to(time, &block)
+  travel_back
```

Comment:
Are you reviewing an old version of the code? I renamed that method, now it's called `safe_travel_to` and it must `travel_back` before calling `travel_to` to avoid nested time traveling, which `TimeHelpers` doesn't allow.